python/pyspark/sql/dataframe.py

```diff
@@ -1980,6 +1980,29 @@ def withColumn(self, colName, col):
         assert isinstance(col, Column), "col should be Column"
         return DataFrame(self._jdf.withColumn(colName, col._jc), self.sql_ctx)
 
+    @ignore_unicode_prefix
+    def withColumns(self, colNames, cols):
```

Comment:
It looks like the Scala impl has this method, but it's not exposed. Instead of adding it here, can you just use two calls to `withColumn` above?